src/services/transaction/TransactionMaterialService.ts

```diff
@@ -13,7 +13,8 @@ export class TransactionMaterialService extends AbstractService {
 		hash: BlockHash,
 		noMeta: boolean
 	): Promise<ITransactionMaterial> {
-		const api = await this.ensureMeta(hash);
+		// const api = await this.ensureMeta(hash);
```

Comment:
remove

Reply:
Not sure why github showed this diff to you. This PR fixed these: https://github.com/paritytech/substrate-api-sidecar/pull/279/files, and all that got merged into master and then here prior to your review.